roles/test/tasks/test-importer.yml

```diff
@@ -0,0 +1,51 @@
+---
+- name: include test data
+  include_role:
+    name: sample-data
+  vars:
+    sample_role_purpose: test
+    sample_fortigate_name: fortigate_test
+    sample_config_user: fworchtest
+    sample_config_user_home: "/home/{{ sample_config_user }}"
+    import_sample_server: localhost
+
+- name: make test import
+  command: "./fworch-importer-single.pl mgm_id=1"
```

Comment:
should probably be something like mgm_name={{ sample_fortigate_name }} to import the correct system

Reply:
introduced a global variable containing the name